src/components/general/ErrorMessage/styles.less

```diff
@@ -0,0 +1,49 @@
+@import "~styles/variables.less";
+
+
+
+.container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+
+    .messageContainer {
+        position: relative;
+        top: 0; left: 0; bottom: 0; right: 0;
+        display: flex;
+        align-content: center;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        background: rgba(255, 255, 255, .8);
```

Comment:
var(--colors-something)?

Reply:
There was no background color was not in the design sheet so it was removed